lib/phlex/testing/capybara.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "capybara"
+
+module Phlex::Testing::Capybara
+	include Phlex::Testing::SGML
+
+	def self.included(mod)
+		if defined?(Minitest::Test) && Minitest::Test > mod
+			require "capybara/minitest"
+			include Capybara::Minitest::Assertions
+		end
+
+		if defined?(RSpec::Core::ExampleGroup) && RSpec::Core::ExampleGroup > mod
+			require "capybara/rspec"
+			include Capybara::RSpecMatchers
+		end
+	end
+
+	attr_reader :page
+	alias_method :component, :page
```

Comment:
We need to have `page` for the built in capybara matchers, but you may want to refer to it as `component`.